# Undo (mostly) pick and place tweaks, as on simulated robots there are not backlash
arm_ctrl_ref_frame: arm_base_link
vertical_backlash_scale: 0.45
vertical_backlash_delta: -0.005
fall_short_distance_delta: 0.005
gripper_asymmetry_yaw_delta: -0.00001
